MMM-plex-recently-added.js

```diff
@@ -102,8 +102,13 @@ Module.register('MMM-plex-recently-added', {
 
   _getThumbUrl(item) {
     var thumbKey = 'thumb';
-    if (item.type === 'episode') {
+    if (item.type === 'episode' | item.type === 'season') {
+		if (item.grandparentThumb){
```

Comment:
Do we want the `grandparentThumb` to take precedence over the `parentThumb`? 

could it be something like:
```
var thumb;
if (item.parentThumb) {
	thumb = item.parentThumb;
} else if (item. grandparentThumb) {
	thumb = item.grandparentThumb;
}
```

or the other way around?

Reply:
grandparentThumb can take precedence. Just need to check for parentThumb if grandparentThumb does not exist.